packages/bierzo-wallet/src/routes/paths.ts

```diff
@@ -4,6 +4,8 @@ export const TRANSACTIONS_ROUTE = "/transactions";
 export const BALANCE_ROUTE = "/balance";
 export const CONFIRM_TRANSACTION_ROUTE = "/confirm-transaction";
 export const ADDRESSES_ROUTE = "/addresses";
-export const REGISTER_PERSONALIZED_ADDRESS_ROUTE = "/register-personalized-address";
+export const REGISTER_IOVNAME_ROUTE = "/register-iovname";
+export const REGISTER_STARNAME_ROUTE = "/register-starname";
+export const REGISTER_NAME_ROUTE = "/register-name";
```

Comment:
I was thinking about URL naming convention. What do you think if we will place it in order "noun"-"verb". In this case it should be `/starname-register`. Or even better `/starname/register`.

Reply:
Great, I like the last option since it will open up other operations like `/starname/renew` or whatever.